cps/CP78.md

```diff
@@ -0,0 +1,128 @@
+Update: It appears that the release CI pipeline is not working as expected. This will be fixed by the next release.
+
+## Centrifuge 1024 - Release Notes Highlights
+BREAKING: Upgrade to Polkadot v0.9.43 (#1577, #1622, #1626, #1640)
+See below for details
+Feat: Add transfer allowlist pallet for local (via balances) and xcm transfers (via xtokens) (#1621)
+Allows setting a filter for transfer destinations
+Feat: Debt transfer (#1615)
+Feat: InvestmentPortfolio API (#1608)
+Liquiditypools:
+Add compute_at to UpdateTrancheTokenPrice (#1598)
+Add restriction_set field to AddTranche message (#1610)
+CI: Rewrite & Improvements (#1551, #1611, #1619, #1624, #1625, #1628, #1636)
+Migrations
+No migrations for this release.
+
+### Breaking Changes
+The following changes are all inherited from upgrading from Polkadot v0.9.38 to Polkadot v0.9.43
+
+The balances::transfer call will soon™️ be deprecated. It is replaces with balances::transfer_allow_death (paritytech/substrate#12951)
+Switch from balance locks and reserves to balance freezes and holds (paritytech/substrate#12951)
+Require existential deposit for all currencies except for tranche tokens (#1596, #1577)
+The amount for each currency is taken from the existential_deposit field in the corresponding Asset Registry entry
+An ExistentialDeposit of zero no longer works. If you need to allow any account to have a zero balance and retain associated account data, you must introduce a pallet which allows an unpermissioned frame_system::Pallet::<Runtime>::inc_providers(). (paritytech/substrate#12951)
+
+### WASM
+
+Since the latest srtool does not produce a WASM report due to this issue, the following WASM information is provided by subwasm v0.16.1 which is internally used by srtool as well.
+
+```
+🏋️ Runtime size:		1.754 MB (1,839,185 bytes)
+🗜 Compressed:			Yes, 78.34%
+✨ Reserved meta:		OK - [6D, 65, 74, 61]
+🎁 Metadata version:		V14
+🔥 Core version:		centrifuge-1024 (centrifuge-1.tx2.au1)
+🗳️ system.setCode hash:		0xb84978e151aa721326b911992d7789d8a8f3b4c21c40185f86a3cb78b8fcd37f
+🗳️ authorizeUpgrade hash:	0xe6073dcc2eef6f734d2d5d946b1e1f456d8c1f449a63bc3dae9e540aea546c66
+#️⃣ Blake2-256 hash:		0xe9ecfd39bfd01a658ba28344963c4206863e81119b361f497538a164c2ef5a97
+📦 IPFS:			https://www.ipfs.io/ipfs/QmZRejYou7do4JpJcMsYM5L2XodbcFx4tTsMsjquiswCbt
+```
+
+**Docker Image**: [`centrifugeio/centrifugechain:main-ccbc3a1-23-12-07`](https://hub.docker.com/layers/centrifugeio/centrifuge-chain/main-ccbc3a1-23-12-07/images/sha256-3526dedf85a477e542e024a9db90d16148df787ac0043791de07835afe8554c6?context=explore)
+**Full Changelog**: [v0.10.34...v0.10.35](https://github.com/centrifuge/centrifuge-chain/compare/v0.10.34...v0.10.35)
+**Release Notes** [v0.10.35](https://github.com/centrifuge/centrifuge-chain/releases/tag/v0.10.35)
+
+
+## Governance process for this proposal
+1. Council Motion + fast-track
+2. Referendum vote open for 28,800 blocks (~4 days)

```

Comment:
4½ days?

Reply:
28,800 blocks this is 4 days but please also consider that we need to find a Councillor who will submit it + reach 5 votes.

Reply:
Referendum vote open for 32,400 blocks (~4.5 days)
changed